### PR TITLE
maven release everything but zanata-test-war and functional-test

### PR DESCRIPTION
**Pull request by definite**
This change is [Reviewable](https://reviewable.io/reviews/zanata/zanata-scripts/11)

### CODE REVIEW

File: zanata-maven-release [hunk 1]
```diff
@@ -296,7 +296,7 @@ case $RepoName in
 	## List all artifacts that will be released in sonatype
 	## as well as zanata-frontend, which is required for MEAD
 	MavenTagProjects+=( -pl build-tools,parent)
-	NexusReleaseProjects+=( -pl build-tools,parent,api,api/zanata-common-api,common,common/zanata-adapter-glossary,common/zanata-adapter-po,common/zanata-adapter-properties,common/zanata-adapter-xliff,common/zanata-common-util,client,client/stub-server,client/zanata-cli,client/zanata-client-commands,client/zanata-maven-plugin,client/zanata-rest-client,server/zanata-frontend )
+	NexusReleaseProjects+=( -pl '!server/zanata-test-war,!server/functional-test' )
```
Comment (seanf) [inline zanata-maven-release:299]: Much better! How did the old one get through review? :-)

File: zanata-maven-release [hunk 2]
```diff
@@ -324,6 +324,6 @@ run_command sed -i -e 's|<tag>.*</tag>|<tag>HEAD</tag>|' $(find . -name pom.xml
 run_command ./mvnw "${MavenOptionArray[@]}" "${MavenTagProjects[@]}" versions:set -DnewVersion=$DevelopmentVersion
 run_command ./mvnw -N scm:checkin -Dmessage="chore(version): new snapshot version ${DevelopmentVersion}" -DpushChanges=true
 
-## Push the tags
-run_command git push --tags
+## Push the relevant tags
+run_command git push --follow-tags origin $WorkingBranch
```
Comment (seanf) [inline zanata-maven-release:328]: Were there some unwanted tags?

**Review by seanf — APPROVED** [review]

File: zanata-maven-release (hunk 2 above)
Comment (definite) [reply zanata-maven-release:328]: Say, if you have removed the previous rc and alpha from GitHub, but have not change the local.
`git push --tags` brings back all of them.

Comment (seanf) [reply zanata-maven-release:328]: But why are we removing existing tags? I wouldn't think Maven would be coping with that very well.

Comment (definite) [reply zanata-maven-release:328]: maven does not care the git tag. And we did remove and re-tag the 4.0.0, 4.1.0, 4.1.1

And this, does confuse some of our git clone.

Comment (seanf) [reply zanata-maven-release:328]: What I mean is that Maven doesn't really like it when you cut the same release more than once. I'm not surprised it confuses git clones. Rewriting history often leads to trouble.

We should avoid retagging releases. The Maven way is just to increment the version and make another release, untidy though it is. Better to be untidy than mutable!

**Pull request merged**